src/extensions/initializeHitherInterface.ts

```diff
@@ -0,0 +1,316 @@
+import axios from 'axios'
```

Comment:
This code is now used by both the jupyter and web gui.

Reply:
it's not new code... just moved around and repackaged in initializeHitherInterface